src/resources/Projects/ProjectInterfaces.ts

```diff
@@ -29,10 +29,34 @@ export const projectResourceTypes = [
 export type ProjectResourceType = (typeof projectResourceTypes)[number];
 
 export enum ProjectType {
+    CaseDeflection = 'CASE_DEFLECTION',
     Commerce = 'COMMERCE',
+    CommunityPortal = 'COMMUNITY_PORTAL',
+    CorporateWebsite = 'CORPORATE_WEBSITE',
+    HrEmployeePortal = 'HR_EMPLOYEE_PORTAL',
+    Intranet = 'INTRANET',
+    InProductSearch = 'IN_PRODUCT_SEARCH',
+    ItEmployeePortal = 'IT_EMPLOYEE_PORTAL',
     Other = 'OTHER',
+    SupportExperience = 'SUPPORT_EXPERIENCE',
+    SupportPortal = 'SUPPORT_PORTAL',
+
+    /**
+     * @deprecated
+     * Use SupportPortal instead
```

Comment:
I think you could link this bit of text with the deprecated tag. What do you think?

```suggestion
     * @deprecated Use SupportPortal instead
```